.github/workflows/pkgdown.yaml

```
@@ -1,46 +1,46 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
   push:
-    branches: main
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
```

I think their suggestion makes sense to cover all cases, but we only need main I believe

```suggestion
    branches: main
  pull_request:
    branches: main
```